### PR TITLE
97 fix python api client examples

### LINKED ISSUE #97: Python examples using the `Vega-API-client` do not work
The python examples that use the python-SDK, such as [this](https://github.com/vegaprotocol/sample-api-scripts/blob/master/get-markets-and-market-data/get-markets-and-marketdata-with-Vega-API-client.py), no longer work since the data-node split. `requirements.txt` is pegged against `Vega-API-client==0.38.0` when the current latest SDK is at `0.47.0`

The sample scripts should be updated/reqritten so that they are working again.

### PR DESCRIPTION
**Pull request by wwestgarth**
closes #97 

Almost all the changes are just due to the protos having shuffled around after the data node split.

### CODE REVIEW

File: submit-amend-cancel-orders/submit-amend-cancel-orders-with-Vega-API-client.py [hunk 1]
```diff
@@ -200,14 +199,14 @@
...
-order_id_request = vac.api.trading.OrderByIDRequest(order_id=orderID)
+order_id_request = vac.data_node.api.v1.trading_data.OrderByIDRequest(order_id=orderID)
 response = data_client.OrderByID(order_id_request)
 
 orderID = response.order.id
 orderPrice = response.order.price
 orderSize = response.order.size
-orderTif = helpers.enum_to_str(vac.vega.Order.TimeInForce, response.order.time_in_force)
-orderStatus = helpers.enum_to_str(vac.vega.Order.Status, response.order.status)
+orderTif = helpers.enum_to_str(vac.vega.veag.Order.TimeInForce, response.order.time_in_force)
```
Comment (edd) [inline submit-amend-cancel-orders/submit-amend-cancel-orders-with-Vega-API-client.py:208]:
```suggestion
orderTif = helpers.enum_to_str(vac.vega.vega.Order.TimeInForce, response.order.time_in_force)
```

**Review by edd — APPROVED** [review]
Approved, as long as you accept my typo fix!

File: README.md [hunk 2]
```diff
@@ -40,7 +40,9 @@ And you're good to go. Now choose a sample program to run from the following (se
 | Vega/blockchain time  | bash, python3 | Vega node [GraphQL, REST, gRPC] | / [vega-time](vega-time) |
 | Streaming events | bash, python3 | Vega node [GraphQL, gRPC] | / [stream-events](stream-events) |
 | Fees and margins estimation | bash, python3 | Vega node [REST, gRPC] | / [fees-margins-estimation](fees-margins-estimation) |
-| Propose, vote and enact new markets | bash, python3 | Vega node [REST] | / [propose-markets](propose-markets) |
+| Propose, vote and enact new markets | python3 | Vega node [REST] | / [propose-markets](propose-markets) |
+| Propose, vote and enact netparam update | python3 | Vega node [REST] | / [propose-markets](propose-netparam) |
```
Comment (candida-d) [inline README.md:44]:
```suggestion
| Propose, vote and enact network parameters update | python3 | Vega node [REST] | / [propose-markets](propose-netparam) |
```

File: submit-order/submit-order-with-Vega-API-client.py [hunk 3]
```diff
@@ -55,92 +56,66 @@
...
-        ),
-    ),
-)
-print(f"Request for SubmitTransaction: {request}")
-submittx_response = tradingcli.SubmitTransaction(request)
-# :submit_tx__
-assert submittx_response.success
-print("All is well.")
+
+print("Signed pegged order and sent to Vega")
```
Comment (candida-d) [inline submit-order/submit-order-with-Vega-API-client.py:120]: Question - is this a pegged order?

**Review by candida-d — CHANGES_REQUESTED** [review]
I've made one suggested change - to spell out network parameters, and asked a question about one console output. Thanks for making these changes Will!

File: submit-order/submit-order-with-Vega-API-client.py (hunk 3 above)
Comment (wwestgarth) [reply submit-order/submit-order-with-Vega-API-client.py]: Nope. Updated.

**Review by candida-d — APPROVED** [review]
LGTM!

**Pull request merged**